opennms-services/src/main/java/org/opennms/netmgt/collectd/Collectd.java

```diff
@@ -1470,7 +1432,7 @@ public Scheduler getScheduler() {
     /**
      * <p>setCollectorConfigDao</p>
      *
-     * @param collectdConfigFactory a {@link org.opennms.netmgt.dao.api.CollectorConfigDao} object.
+     * @param collectdConfigFactory a {@link org.opennms.netmgt.config.CollectdConfigFactory} object.
```

Comment:
I'd opt in for either not changing the javadoc or remove it completly, as it is not adding any more useful information.